packages/cli/src/models/compiler/Compiler.js

```diff
@@ -1,13 +1,13 @@
 import { exec } from 'child_process'
 import { FileSystem, Contracts, Logger } from 'zos-lib'
-import TruffleConfig from '../initializer/truffle/TruffleConfig'
+import Truffle from '../initializer/truffle/Truffle'
```

Comment:
I think that using an external library name (Truffle) for a zos local class/object could be somewhat confusing. I don't have any other name in mind, though.